src/plugins/screenshot/Screenshot.cc

```diff
@@ -0,0 +1,273 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#include "Screenshot.hh"
+
+#include <string>
+
+#include <ignition/common/Console.hh>
+#include <ignition/common/Filesystem.hh>
+#include <ignition/common/Image.hh>
+#include <ignition/plugin/Register.hh>
+#include <ignition/rendering/Camera.hh>
+#include <ignition/rendering/RenderEngine.hh>
+#include <ignition/rendering/RenderingIface.hh>
+#include <ignition/rendering/Scene.hh>
```

Comment:
The last 2 Windows warnings are coming from `ign-rendering`. I'm not sure why they don't show up on upstream CI. In any case, we've been suppressing those warnings too, like this:

https://github.com/ignitionrobotics/ign-gui/blob/10affb2b101c333f0e4c194e79444ba9179f6b61/src/plugins/scene3d/Scene3D.cc#L35-L51

Reply:
Thank you! 8dd1d124427d92306da5fec432db4cfbc07deefd